xrpl/ledger-entry-types/amm.go

```diff
@@ -88,12 +119,29 @@ func (s *AMMAuctionSlot) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
-type AMMAuthAccount struct {
+// ---------------------------------------------
+// AuthAccounts Object
+// ---------------------------------------------
+
+type AuthAccounts struct {
```

Comment:
I would move this types to the top of the file since it is more easy to read and find the AMM field types declaration